composer.json

```diff
@@ -7,7 +7,8 @@
     "require": {
         "php": "~7.4",
         "sylius/sylius": "^1.8",
-        "phpstan/phpstan-webmozart-assert": "^0.12.7"
+        "phpstan/phpstan-webmozart-assert": "^0.12.7",
+        "monsieurbiz/sylius-settings-plugin": "^1.0.0"
```

Comment:
You need to install the plugin in the test application.

Reply:
i create a new PR to update the test application, based on your WIP PR